fast_td3/fast_td3/actors/gnn/egcl.py

```
@@ -0,0 +1,148 @@
+        hidden_nf,
+        edges_in_d,
+        act_fn=nn.SiLU(),
+        residual=True,
+        attention=False,
+        normalize=False,
+        coords_agg="mean",
+        tanh=False,
+        edge_coords_nf=1,
+    ):
+        super(E_GCL, self).__init__()
+        input_edge = input_nf * 2
```

Ruff (0.14.4)

Local variable `input_edge` is assigned to but never used

Remove assignment to unused variable `input_edge`

(F841)